local/project/config.go

```diff
@@ -78,16 +76,18 @@ func NewConfigFromDirectory(logger zerolog.Logger, homeDir, projectDir string) (
 
 	// first consider project configuration files in this specific order
 	for _, suffix := range []string{".dist.yaml", ".yaml", ".override.yaml"} {
-		fileConfig, err := newConfigFromFile(filepath.Join(projectDir, ConfigFilePrefix+suffix))
-		if errors.Is(err, os.ErrNotExist) {
-			continue
-		} else if err != nil {
-			return nil, err
-		} else if fileConfig == nil {
-			continue
-		}
+		for _, configFilePrefix := range []string{".symfony.local", "symfony-cli", ".symfony-cli"} {
```

Comment:
I think I would move this as the outerLoop. This way we can break as soon as we have a a match on one prefix and not let users mix everything together